### DISCUSSION TITLE
SMEM-only implementation?

### DISCUSSION THREAD

**Discussion by evolgenomics** [ITBE-Lab/MA#13] [Q&A]
Hi MA Team - Markus & Arne,

I'm playing around with your Modular Aligner software. It looks well built, and does have all kinds of flexibilities I'm looking for. However, I may need a bit more documentation to build the command in the python environment. I was trying to follow the MA-EVAL repo, but was also missing explicit examples there. 

In particular, I'm hoping to evaluate the difference between just SMEM+SoC/Chaining vs. the full steps 1-3 with dynamic programming. 

Could you provide some use case examples or a short guide?

Thanks.

**Comment by MarkusRainerSchmidt** [comment]
Hi,

We are happy to help, but could you please be a bit more specific about where you run into trouble?

The point of MA-EVAL is to produce the evaluations and figures in the "Accurate high throughput alignment via line sweep-based seed processing" manuscript.
As such, I am not really sure if it still works with the latest versions of MA and would recommend using Version 0.1.2-alpha (the version that was used to create the figures of the manuscript).

For your analysis:
The analysis uses the command line version of MA, so you would need to add a parameter to the command-line tool to skip DP.
Then, if all the analyzed aligners (see e.g. create_accuracy_graph.py line 34 and onwards) are callable on the command line, you should be able to run the individual python files producing the figures.

Sorry for being so vague, 
if you have some more specific questions please feel free to ask!

Markus
